### DISCUSSION TITLE
[Feature] use Renovate or Dependabot?

### DISCUSSION THREAD

**Discussion by uhthomas** [immich-app/immich#2738] [Feature Request]
### The feature

It would be nice to keep Immich dependencies up to date automatically.

### Platform

- [X] Server
- [X] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
We'd been putting this off for too long :) It's set to go in https://github.com/immich-app/immich/pull/2739, we'll kick that off after the next release (because it's going to make a lot of PRs :P)

**Comment by martabal** [comment]
Can we exlude some dependencies ? The AIO image is based on latest ubuntu non LTS version. sharp v32.0 requires libvips `v8.14.2`, but the latest libvips version for ubuntu is `8.14.1-3`. Moreover, sharp v32.0 has been released in March 2023 and the only sharp version for alpine 3.17 is `v8.13.3`. It seems latest sharp versions use latest libvips packages available on alpine edge.
